apps/backend/test/e2e/auth.e2e-spec.ts

```diff
@@ -0,0 +1,131 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { ConfigModule } from '@nestjs/config';
+
+import * as request from 'supertest';
+
+import {
+  closeInMongodConnection,
+  rootMongooseTestModule,
+} from '../utils/in-memory-mongo';
+import { userMock } from '../mocks/user';
+import { AuthModule } from '../../src/auth/auth.module';
+import { AuthService } from '../../src/auth/auth.service';
+import { UserModule } from '../../src/user/user.module';
+
+const setup = async () => {
+  const moduleFixture: TestingModule = await Test.createTestingModule({
+    imports: [
+      rootMongooseTestModule(),
+      UserModule,
+      AuthModule,
+      ConfigModule.forRoot({ isGlobal: true }),
+    ],
+  }).compile();
+
+  const app = moduleFixture.createNestApplication();
+  const service = moduleFixture.get<AuthService>(AuthService);
+
+  await app.init();
+
+  return {
+    app,
+    service,
+  };
+};
+
+const tomorrow = () => {
+  const date = new Date();
+  return date.setDate(date.getDate() + 1);
+};
+
+describe('Authentication (e2e)', () => {
+  it('/POST auth/signUp', async () => {
+    const { app } = await setup();
+
+    const { body } = await request(app.getHttpServer())
+      .post('/api/v1/auth/signUp')
+      .send(userMock)
+      .expect(201);
+
+    expect(body).toStrictEqual({
+      message: 'User created successfully',
+    });
+
+    await app.close();
+  });
```

Comment:
Not a big deal but you nest describe blocks to make the expected output explicit:

```js
describe('/POST auth/signUp', () => {
 it('returns a successful message', () => { ... })
})
```